### DISCUSSION TITLE
Welcome to LibSassBuilder Discussions!

### DISCUSSION THREAD

**Discussion by johan-v-r** [johan-v-r/LibSassBuilder#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jn-ifm** [comment]
Hi, do you plan to move forward to .net 6 in the near future with this project to get rid of the build requirement of installed .net5.0?

**Reply by johan-v-r** [reply]
Hi - yeah once it's released to GA I'll update this.

**Reply by jn-ifm** [reply]
Great and thx for this info :)
